bindings/nodejs/lib/types/block/output/feature.ts

```diff
@@ -113,12 +114,15 @@ class BlockIssuerFeature extends Feature {
     /**
      * The Block Issuer Keys.
      */
-    readonly publicKeys: Set<HexEncodedString>;
+    readonly blockIssuerKeys: Set<Ed25519BlockIssuerKey>;
```

Comment:
Why is it a set of specific ed25519 keys ?

Reply:
Because it can't be anything else?

Reply:
For now yeah, shouldn't we still have a generic to be future proof ?

Reply:
Ah yeah if that's not more annoying, updated